AUTHORS

```diff
@@ -8,8 +8,11 @@
 #
 # Please keep the list sorted alphabetically.
 
+Aashish Gaba <aashishgaba097@gmail.com>
 Abhay Raizada <abhsag24@gmail.com>
 Abhijit Suresh <abhijit5893@gmail.com>
+Abhishek Arya <abhishekarya286@gmail.com>
+acash mkj <31628272+acashmkj@users.noreply.github.com>
```

Comment:
Should this be capitalized? Seems inconsistent with other names. Also, no real email address?

Reply:
Yeah, I noticed that. I don't have anything else, though, so it'll need to go in as-is.